packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -328,17 +328,6 @@ const ReviewVotingPage = ({classes}: {
     [sortedPosts]
   )
 
-  if (!currentUserCanVote(currentUser)) {
```

Comment:
All users can now see the /reviewVoting page because it's also where reviews are shown, and all users can participate in reviews.